graylog2-web-interface/src/views/pages/NewSearchPage.jsx

```diff
@@ -44,7 +45,18 @@ class NewSearchPage extends React.Component<Props, State> {
   }
 
   componentDidMount() {
-    ViewActions.create(View.Type.Search).then(() => this.setState({ loaded: true }));
+    const { location, loadingViewHooks, executingViewHooks } = this.props;
+    const { query } = location;
+    processHooks(
+      ViewActions.create(View.Type.Search).then(({ view }) => view),
+      loadingViewHooks,
+      executingViewHooks,
+      query,
+    ).then(
+      () => this.setState({ loaded: true }),
+    ).catch(
+      error => UserNotification.error(`Executing search failed with error: ${error}`, 'Could not execute search'),
```

Comment:
When the url params range type is `keyword` and the provided keyword is invalid, the search page does not render. On the old page we displayed an inline error message, but I think it's also okay to display this type of error message?

Reply:
For now, yes.